api/src/main/java/org/openmrs/api/db/hibernate/HibernateContextDAO.java

```diff
@@ -294,14 +292,12 @@ private Integer getUsersLoginAttempts(User user) {
 	public void openSession() {
 		log.debug("HibernateContext: Opening Hibernate Session");
 		if (TransactionSynchronizationManager.hasResource(sessionFactory)) {
-			if (log.isDebugEnabled()) {
-				log.debug("Participating in existing session (" + sessionFactory.hashCode() + ")");
-			}
```

Comment:
both the logs in this file have the variable enclosed in `()`. So it should be

`log.debug("Participating in existing session ({})", sessionFactory.hashCode());`